packages/fxa-auth-server/lib/senders/emails/templates/unblockCode/index.stories.ts

```diff
@@ -3,19 +3,18 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { Meta } from '@storybook/html';
-import { buildStory } from '../../storybook-email';
+import { MOCK_LOCATION } from '../../partials/location/mocks';
+import { storyWithProps } from '../../storybook-email';
 
 export default {
   title: 'Emails/unblockCode',
 } as Meta;
 
-const createStory = buildStory(
+const createStory = storyWithProps(
   'unblockCode',
   'Sent to verify an account sign-in via code.',
```

Comment:
Just noticed, this documentation for `unblockCode` could probably be more descriptive, like, `'Sent to verify a blocked account sign-in via code.'` if that's correct. But even then, when is an account blocked or why would they get this instead of `verifyLoginCode`?

Reply:
That's fair. I based these descriptions off of that [original spreadsheet](https://docs.google.com/spreadsheets/d/11xFyTRpYkoYcUc3Ac1x6z976AvMpgRaV6blGK9Q838g/edit#gid=1031776270), where they are pretty short and to the point. So that's what I kept doing here: short and to the point. I was actually going to maybe suggest that we shorten your description in the `verifyPrimary` story a tad to be consistent with the rest. Or we can go back in and update all the existing ones to be even more descriptive, but in some cases that might not be totally possible, and I don't think it should take significant time to hunt down and compose lengthy descriptions given the volume of templates we have left to convert.

Reply:
Hmm I feel the entire point of documenting these is to disambiguate when certain emails are sent/received though. Reading this, I don't know when a user would receive the `unblockCode` email over a more standard `verifyLoginCode` email.  I'm fine with simpler descriptions so long as we leave that follow up to go in and edit these further, otherwise I think the documentation description should be as clear as we can be while also not making it like multiple sentences or something. 🤔